### PR TITLE
Beacon integration

### PR DESCRIPTION
**Pull request by jderrett**
This adds support for the internal Beacon integration.

### CODE REVIEW

File: services/beacon.rb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+      # Property bag is a JSON blob that gets stored in Beacon along with the alert
+      body[:property_bag] = {
+        alert_id: "#{payload[:alert][:id]}",
+        alert_name: payload[:alert][:name],
+        alert_description: payload[:alert][:description].blank? ? payload[:alert][:name] : payload[:alert][:description]
+      }
+      unless payload[:alert][:runbook_url].blank?
+        body[:property_bag][:runbook_url] = payload[:alert][:runbook_url]
+      end
```
Comment (mheffner) [inline services/beacon.rb]: No details on the violated conditions are sent?

Comment (leecalcote) [reply services/beacon.rb]: Outside of the url / id of the specific alert instance, is the id of the alert definition available to be sent as well?

Comment (jderrett) [reply services/beacon.rb]: @mheffner thanks for the reminder, I'm working on adding violated conditions now.

@leecalcote the alert definition is included here https://github.com/appoptics/appoptics-services/pull/12/files/e343a77c7d197082379bc1f9e564ab57dae01558#diff-9542cb0142c71bf67c02763741be5b97R21 which should be unique.

Comment (leecalcote) [reply services/beacon.rb]: @jderrett got it. Good deal.

**Pull request merged**